cmd/dlt/autoscaler/cmd.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
```

Comment:
![https://media.giphy.com/media/GcDtLf4RAdiRG/giphy.gif](https://media.giphy.com/media/GcDtLf4RAdiRG/giphy.gif)

Reply:
Hahahaha yeah thanks ;)

Reply:
comment of the year :rofl: